src/openrct2/libopenrct2.vcxproj

```diff
@@ -1028,4 +1028,4 @@
     </ClCompile>
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-</Project>
+</Project>
```

Comment:
Unrelated change, revert.

Reply:
I always assumed these were sort of... vacuumed up. They haven't been called out before. I'll change it.